layers/meta-balena-fsl-arm/recipes-kernel/linux/linux-fslc-imx_%.bbappend

```diff
@@ -0,0 +1,25 @@
+FILESEXTRAPATHS:append := ":${THISDIR}/${PN}"
+
+inherit kernel-resin
+
+# Latest known revision as of now is 49cd2d27fefdbaa79f32df96c4672433d9d45590
+# and is set in meta-freescale-third-party linux-boundary recipe.
+
+# Disable commit SHA in kernel version string
+SCMVERSION="n"
+
+SRC_URI:append = " \
+	file://0001-disable-audio-codecs.patch \
+        file://0002-disable-caam-jr2.patch \
+        file://enable-iwlwifi.cfg \
+"
+
+BALENA_CONFIGS:append = " optimize-size disable_apparmor"
+BALENA_CONFIGS[optimize-size] = " \
```

Comment:
Since this is a new machine in which you enlarged the rootfs and boot partition sizes we can skip optimizing for size